code/object/objectsnd.cpp

```diff
@@ -612,7 +618,7 @@ void obj_snd_do_frame()
 						break;
 
 					default:
-						Int3();	// get Alan
+						UNREACHABLE("Unhandled object type for persistent sound; get Alan");
```

Comment:
You can leave the get Alan, of course, but make sure a person only loosely connected to us will know to report it to FSO.  Something along the lines of "get Alan -- I mean, an SCP coder!"  

Also, if we have some weird object type proliferating here, we should know what type it is.

Reply:
That kinda takes the fun out of it, but changed.